packages/perseus/src/widgets/interactive-graphs/mafs-graph.tsx

```diff
@@ -40,6 +40,8 @@ const renderGraph = (props: {
             return <PolygonGraph graphState={state} dispatch={dispatch} />;
         case "point":
             return <PointGraph graphState={state} dispatch={dispatch} />;
+        case "circle":
+            throw new Error("the circle graph type is not yet implemented");
```

Comment:
I added the circle type as a kind of test case to make sure the refactored code will be flexible in the ways we need it to be. But since circle graphs are not actually implemented yet, I should probably take this out (and remove the `circle` graph state type) before landing this PR.